jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/RemoteExceptionMapper.java

```diff
@@ -38,10 +38,23 @@
     @Override
     public Response toResponse(RemoteException exception) {
         String errorId = UUID.randomUUID().toString();
-        log.error("Error handling request {}", SafeArg.of("errorId", errorId), exception);
 
-        SerializableError error = exception.getRemoteException();
         Status status = Status.fromStatusCode(exception.getStatus());
+
+        // here in the client, log responses that indicate client error (4xx) at higher level than server error (5xx)
+        if (Status.Family.CLIENT_ERROR.equals(status.getFamily())) {
+            log.error("Received response status code {} from server handling request. errorId: {}",
+                    SafeArg.of("statusCode", status.getStatusCode()),
+                    SafeArg.of("errorId", errorId),
+                    exception);
+        } else {
+            log.info("Received response status code {} from server handling request. errorId: {}",
```

Comment:
I set to INFO to match https://github.com/palantir/http-remoting/pull/458/files#diff-bca185972ae917cdb714c8c40eacb218R58 but @mahmoudm also suggested WARN